CONTRIBUTING.adoc

```diff
@@ -0,0 +1,25 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/code-ready/crc/issues[GitHub issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+. Create an issue describing your proposed change to the repository.
+. The repository owners will triage and respond to your issue promptly.
```

Comment:
This isn't a step in the process that the reader needs to perform. We can remove the "`. `" before this sentence so that it's part of the first step in the process (as a result of creating an issue).